src/System.Console/tests/ConsoleEncoding.cs

```diff
@@ -21,7 +21,7 @@ public static IEnumerable<object[]> InputData()
     }
 
     [Theory]
-    [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Issue https://github.com/dotnet/corefx/issues/18220")]
+    [ActiveIssue("https://github.com/dotnet/corefx/issues/18220", TargetFrameworkMonikers.UapAot)]
```

Comment:
Do we not have an overload that just takes the issue number, like we do when a target platform isn't supplied?